docs/source/intro/install_conda.rst

```diff
@@ -84,6 +84,8 @@ This is the recommended method for users on shared systems and systems where the
        Last name: name
        Institution: where-i-work
 
+   If AiiDA has any trouble autodetecting the PostgreSQL setup, you can check the optional manual setup explained in the :ref:`troubleshooting section<intro:troubleshooting:installation:quicksetup>`.
```

Comment:
I think this was a misunderstanding. I did not mean to completely remove the admonition container, just its title (if possible).

Reply:
So, I found [a way](https://stackoverflow.com/questions/61435028/how-to-create-boxed-text-without-title-in-sphinx) to do it, but the box does not look exactly the same (the orange line where the title would be is thinner and the symbol escapes from it). Let me know if you think it is good or if I should revert to a title.